tests/end2end-tests/custom-gnn/train.py

```diff
@@ -108,8 +108,20 @@ def main(args):
                            help="The name of the label.")
     argparser.add_argument("--num-classes", type=int, required=True,
                            help="The number of classes.")
-    argparser.add_argument("--local_rank", type=int,
-                           help="The rank of the trainer.")
+    if TORCH_MAJOR_VER >= 2:
+        argparser.add_argument(
+                "--local-rank",
+                type=int,
+                default=0,
+                help="The rank of the trainer.",
+                )
+    else:
+        argparser.add_argument(
+                "--local_rank",
+                type=int,
+                default=0,
+                help="The rank of the trainer.",
+                )
```

Comment:
No need to check for PyTorch version, you can just use:

```
parser.add_argument("--local-rank", "--local_rank", type=int, ...)
```

see https://pytorch.org/docs/stable/distributed.html#launch-utility